cmd/mattermost/commands/server.go

```diff
@@ -34,16 +36,42 @@ func init() {
 	RootCmd.RunE = serverCmdF
 }
 
+func loadCustomDefaults() (*model.Config, error) {
+	var customDefaults *model.Config
+	customDefaultsPath := os.Getenv("MM_CUSTOMDEFAULTS")
```

Comment:
Maybe we can name this env var `MM_CUSTOMDEFAULTSPATH` to make it clear is a file path?
```suggestion
	customDefaultsPath := os.Getenv("MM_CUSTOMDEFAULTSPATH")
```

Reply:
Changed it to `MM_CUSTOM_DEFAULTS_PATH` and updated the Release Notes too, please let me know if that's fine or you prefer to have no separation between words

Reply:
`MM_CUSTOM_DEFAULTS_PATH` sounds good to me. Thanks!